optuna_dashboard/ts/components/TrialTable.tsx

```diff
@@ -293,6 +333,47 @@ export const TrialTable: FC<{
             />
           </Box>
         </Grid>
+        {trials[index].state === ("Running" as TrialState) ? (
+          <Grid item xs={12}>
+            <Box margin={1}>
+              <Typography variant="h6" gutterBottom component="div">
+                Trial tell operations
+              </Typography>
+              <form onSubmit={handleSubmit}>
+                <Box margin={1}>
+                  <Stack direction="row" spacing={1}>
+                    {objectiveValues.map((value, i) => (
+                      <TextField
+                        id={`objective-${i}`}
+                        key={`objective-${i}`}
+                        label={`Objective ${i}`}
+                        type="number"
+                        value={objectiveValues[i]}
+                        onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                          handleChangeValue(i, e)
+                        }
+                      />
+                    ))}
```

Comment:
There are multiple TextFields if the study is multi-objective. Could you add form labels to indicate which values should be entered in each field?

Also we can refer `studyDetail.objectiveNames` like the following.
https://github.com/optuna/optuna-dashboard/blob/29233909a2d8ecba4b6e1a16f0cfdb9d76091d74/optuna_dashboard/ts/components/GraphParetoFront.tsx#L62-L64